Tests/conf.json

```
@@ -1195,7 +1195,13 @@
         },
         {
             "integrations": "MimecastV2",
-            "playbookID": "Mimecast test"
+            "playbookID": "Mimecast test",
+            "instance_names": "api1"
```

can we change it to -
```suggestion
            "instance_names": "mimecast_api_v1"
```